packages/web/src/components/user-list/UserList.tsx

```diff
@@ -48,30 +45,64 @@ type ConnectedUserListOwnProps = {
   onNavigateAway?: () => void
 }
 
-type ConnectedUserListProps = ConnectedUserListOwnProps &
-  ReturnType<typeof mapStateToProps> &
-  ReturnType<typeof mapDispatchToProps>
-
-const ConnectedUserList = (props: ConnectedUserListProps) => {
+const ConnectedUserList = ({
```

Comment:
Since this is no longer "connected" can this just be UserList?

Reply:
yes but `UserList` is also the name of the inner actual component.. but doesn't matter i think we can name it the same.

Reply:
@dylanjeffers i moved everything into the same file, how do we feel about that?